mix.exs

```diff
@@ -30,12 +33,11 @@ defmodule Kitto.Mixfile do
      {:plug, "~> 1.2"},
      {:poison, "~> 3.0"},
      {:fs, github: "kittoframework/fs", tag: "v1.9-kitto-1"},
-     {:ex_doc, ">= 0.0.0", only: :dev},
+     {:ex_doc, "~> 0.14", only: :dev},
```

Comment:
@Zorbash oops forgot it 
`mix deps.unlock --all`
`mix deps.get`

maybe adding in a new PR typespecs and behaviours http://elixir-lang.org/getting-started/typespecs-and-behaviours.html